.mockery.yaml

```diff
@@ -0,0 +1,15 @@
+quiet: False
+disable-version-string: True
+with-expecter: True
+mockname: "{{.InterfaceName}}"
+filename: "{{.MockName}}.go"
+outpkg: mocks
+dir: "{{.InterfaceDir}}/mocks"
```

Comment:
I prefer to have one global mocks folder, currently we have:
![image](https://github.com/Dynatrace/dynatrace-operator/assets/1374633/0956df7b-4c84-4993-8f21-cdb54271876d)

Reply:
@0sewa0 it's related to you global rename folders PR ^

Reply:
in some way yes, but I don't have any major againts having it in 1 or multiple places.
- I am leaning a bit towards the "have it in the same folder as the interface", because its nicer to use IMO
	- However when it comes to external dependencies then its kinda given that those have to be in a dedicated folder

If we want to have it in a single folder, then I would have it in `src/testing/mocks`.

Reply:
not pkg/test/mocks?